cross-platform/react-app/src/backend/main.ts

```diff
@@ -9,6 +9,7 @@ import { MobileHost, MobileHostOpts } from "@itwin/core-mobile/lib/cjs/MobileBac
 import { BackendLogParams, getSupportedRpcs } from "../common/rpcs";
 import { IModelHostConfiguration, IpcHost } from "@itwin/core-backend";
 import { BackendIModelsAccess } from "@itwin/imodels-access-backend";
+import { createDefaultClientStorage } from "@itwin/imodels-access-backend/lib/cjs/DefaultClientStorage";
```

Comment:
maybe try 6.0.1

Reply:
I will, since it fixes a problem that I reported. Having said that, it **just** released.